server/pbenchinacan/deploy

```diff
@@ -113,6 +114,7 @@ podman run \
     --volume ${PB_DEPLOY_FILES}/etc/rsyslog.conf:/etc/rsyslog.conf:Z \
     --volume ${PB_DEPLOY_FILES}/etc/rsyslog.d:/etc/rsyslog.d:Z \
     --volume ${PB_SSL_CERT_FILE}:/etc/pki/tls/certs/pbench-server.crt:Z \
+    --volume ${PB_SSL_CA_FILE}:/etc/pki/tls/certs/pbench_CA.crt:Z \
```

Comment:
This is...disappointing.

I understand why this is here:  in a functional test deployment, the Pbench Server needs access to the CA in order to verify the TLS connection to the canned OIDC server during startup.

However, in Staging/Production, we won't need or want this.

The requirement for this is driven by the `pbench-server.cfg` file, right?  And, that's created when the container is built.  So, instead of mapping the CA file into the container, we should copy it into place when the container is built.